arrow-libs/core/arrow-core/src/commonTest/kotlin/arrow/core/test/Laws.kt

```diff
@@ -6,12 +6,14 @@ import io.kotest.core.spec.style.StringSpec
 import io.kotest.core.spec.style.scopes.StringSpecScope
 import io.kotest.core.spec.style.scopes.addTest
 import io.kotest.core.test.TestContext
+import kotlinx.coroutines.test.TestScope
+import kotlinx.coroutines.test.runTest
 
 interface LawSet {
   val laws: List<Law>
 }
 
-data class Law(val name: String, val test: suspend TestContext.() -> Unit)
+data class Law(val name: String, val test: suspend TestScope.() -> Unit)
```

Comment:
**Nit:** I think we can remove `TestScope` here as a receiver, because it's not being used in any of the laws.

```suggestion
data class Law(val name: String, val test: suspend () -> Unit)
```

Reply:
Well spotted - I adapted the changeset accordingly.